cs4660/graph/graph.py

```diff
@@ -113,27 +219,121 @@ def __init__(self):
         self.nodes = []
 
     def adjacent(self, node_1, node_2):
+        """
+        * adjacent(node_1, node_2)
+            - returns true if node_1 and node_2 are
+            directly connected or false otherwise
+        """
+        if node_1 not in self.nodes or node_2 not in self.nodes:
+            return False
+        node1index = self.nodes.index(node_1)
+        node2index = self.nodes.index(node_2)
+
+
+        if self.adjacency_matrix[node1index][node2index] > 0 \
+        or \
+        self.adjacency_matrix[node2index][node1index] != 0:
```

Comment:
I think you can just compare if it is zero to zero rather than checking both greater or not equals.